workspaces/lightspeed/plugins/lightspeed-backend/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@red-hat-developer-hub/backstage-plugin-lightspeed-backend",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public"
+  },
+  "backstage": {
+    "role": "backend-plugin",
+    "pluginId": "lightspeed",
```

Comment:
This should be `@red-hat-developer-hub/backstage-plugin-lightspeed-backend`, right?

Reply:
No, I think this metadata should be the id of the plugin as a whole. eg: When you create a backstage plugin using `yarn new --select backend-plugin` it will just ask you for an id, once you provide it automatically suffix it with `{pluginId}-backend}`